src/components/homePage/Search/__tests__/Search.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import Search from "../Search";
+
+describe("Search component", () => {
+  test("renders Search", () => {
+    render(
+      <Search
+        tryFindFilmBySearch={jest.fn()}
+        tryToFindFilmsByTyiping={jest.fn()}
+        setSearchText={jest.fn()}
+      />
+    );
+    const input = screen.getByPlaceholderText("Search...");
+    expect(input).toBeInTheDocument();
+  });
+
+  test("onchange working", () => {
```

Comment:
rename to "OnChange should be called with correct params "

Reply:
fixed